spec/factories/course_user_achievements.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :course_user_achievement, class: Course::UserAchievement.name do
+    course_user
+    achievement
+    obtained_at "2015-10-11 23:20:07"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.